check/format-incremental

```diff
@@ -133,8 +142,15 @@ if (( only_print == 1 )); then
     args+=("--check" "--diff")
 fi
 
-black "${args[@]}" "${format_files[@]}"
-BLACKSTATUS=$?
+BLACKSTATUS=0
+if (( "${#format_py_files[@]}" )); then
+    black "${args[@]}" "${format_py_files[@]}"
+    BLACKSTATUS=$?
+fi
+if (( "${#format_ipynb_files[@]}" )); then
+    black "${args[@]}" --ipynb "${format_ipynb_files[@]}"
+    BLACKSTATUS=$((BLACKSTATUS || $?))
+fi
```

Comment:
Please revert back.  It works OK if `black` is run without the `--ipynb` option.